### PR TITLE
Fixes with #includes + build file

### PR DESCRIPTION
**Pull request by worksofliam**
I was having an issue with the #includes for the MI prototypes which are usually held in QSYSINC/MIH. 

I changed those includes to use them from their relevant source members so the C source can be compiled from the IFS.

I also included a build file so I am able to download install this repository as a package with Relic, which was my initial goal.

`RELICGET PLOC('https://github.com/WorksOfBarry/iopen/archive/master.zip') PDIR('iopen-master') PNAME(INSTALLIB)`

### CODE REVIEW

**Comment by bobcozzi** [comment]
I don't have RELIC so whatever works for you.  Didn't realize #include had been modified to support the "IFS naming" syntax for Source Members. I know RPG has for years. Good to know. Kind of odd the compile-from-IFS doesn't directly support the legacy syntax, however.

File: build.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+build:
+  //Delete existing source
+  CLRLIB &INTOLIB
+
+  ADDLIBLE LIB(&INTOLIB)
+  
+  //Let's copy all the files
+  CRTSRCPF FILE(&INTOLIB/QCSRC) RCDLEN(112)
+  CPYFRMSTMF FROMSTMF('&DIR/crtstmf.c') TOMBR('/QSYS.lib/&INTOLIB.lib/QCSRC.file/CRTSTMF.mbr') MBROPT(*ADD)
```
Comment (bobcozzi) [inline build.txt]: Do you really mean MBROPT(*ADD) here? Should it be *REPLACE?

Comment (worksofliam) [reply build.txt]: Not sure it really matters actually, seeing as I'm doing a CLRLIB on the library.

**Comment by worksofliam** [comment]
@bobcozzi If you're okay with these changes + the build file, can you merge? The sooner you merge the quicker I can get this the list of packages for the package manager as I'm trying to build this up.

Thanks!

**Comment by bobcozzi** [comment]
To confirm, you're saying you have to QUALIFY the #include source members in order to compile "out of the IFS"? Isn't there a path or something you can set during the build? Doesn't seem very _IBM i_-like to me.

**Comment by worksofliam** [comment]
Instead, so your source stays the same, I could move them to a SPF and then compile from there - then the #include can be how it originally was.

The only reason I changed it is because it didn't like those includes.. it kept thinking that they didn't exist even though we know they do.

**Comment by bobcozzi** [comment]
Did you try the OPTION(*SYSINCPATH) option on CRTBNDC?

Not sure if that'll work, but might be worth a try.

-Bob

On Sep 9, 2016, at 8:21 AM, Liam Allan notifications@github.com wrote:

Instead, so your source stays the same, I could move them to a SPF and then compile from there - then the #include can be how it originally was.

The only reason I changed it is because it didn't like those includes.. it kept thinking that they didn't exist even though we know they do.

**Comment by worksofliam** [comment]
Bob, looks like that fixed it, thanks! I will revert the changes I made and then change the build file. It should be good to go then.

**Comment by worksofliam** [comment]
I will instead open a different pull request to there is less spam.

**Pull request closed without merging**